### ISSUE TITLE
Heracles treemap drill-down reports not shown

### ISSUE THREAD

**Issue by leeevans** [OHDSI/WebAPI#933]
1. Go to public atlas and open cohort definition with generated heracles reports:
http://www.ohdsi.org/web/atlas/#/cohortdefinition/1770148
2. Click on reporting tab
3. In 'Select a Source' choose SYNPUF 5%
4. In ‘Select a Report’ - e.g. choose Condition Eras, Drug Eras etc.
5. Click on Treemap tab
6. click on any box in the Treemap

Expected:
There is a drilldown report for a chosen box(concept). It should include: Prevalence, Prevalence by Month, Type, Age at First Occurrence.

Proposed solution:
Follow the same approach in casting stratums in cohortresults sql that was applied to achilles reports in #904

I encountered this issue while testing with the v2.7.0-rc branch

**Comment by pavgra** [comment]
Please keep in mind the following PR:
https://github.com/OHDSI/WebAPI/pull/960/files

**Comment by anthonysena** [comment]
Note to self: re-test this on current master

**Comment by anthonysena** [comment]
@pavgra - confirmed that this is still an issue on the current master branch. Since your team has worked on this recently, would you mind taking this one? Thanks!

**Comment by anthonysena** [comment]
Re-opening this since I'm still facing difficulties with this report. Please see the cohort & steps to reproduce in https://github.com/OHDSI/Atlas/issues/1872. When clicking on the drilldown report for metformin (which the cohort requires all members to have at index) I get the following display

Here is what is returned from WebAPI's request to : `WebAPI/cohortresults/<source_key>/<cohort_id>/drugera/1503297?refresh=true`:

```
{
	"ageAtFirstExposure": [],
	"lengthOfEra": [{
			"category": "Length of Era",
			"conceptId": 1503297,
			"p10Value": 30,
			"p25Value": 30,
			"p75Value": 286,
			"p90Value": 602,
			"minValue": 0,
			"medianValue": 100,
			"maxValue": 5733
		}
	],
	"prevalenceByGenderAgeYear": [],
	"prevalenceByMonth": []
}
```

I've confirmed that my version of WebAPI includes the fixes from #1126.

**Comment by anthonysena** [comment]
The same behavior also exists for the Condition Eras report:

`WebAPI/cohortresults/<source_key>/<cohort_id>/conditionera/201826?refresh=true`:

```
{
	"ageAtFirstDiagnosis": [],
	"lengthOfEra": [{
			"category": "Length of era",
			"conceptId": 201826,
			"p10Value": 0,
			"p25Value": 0,
			"p75Value": 7,
			"p90Value": 26,
			"minValue": 0,
			"medianValue": 0,
			"maxValue": 1714
		}
	],
	"prevalenceByGenderAgeYear": [],
	"prevalenceByMonth": []
}
```
